src/main/java/gmbs/model/RacingCars.java

```diff
@@ -0,0 +1,53 @@
+package gmbs.model;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class RacingCars {
+
+    private final List<Car> cars = new ArrayList<>();
+    private int headPositionValue;
+
+    public RacingCars(List<String> carNames) {
+        for (String name : carNames) {
+            cars.add(new Car(name));
+        }
+    }
+
+    private void headPositionUpdate(int position) {
+        if (position >= headPositionValue) {
+            headPositionValue = position;
+        }
+    }
+
+    public void updateStatus(NumberGenerator generator) {
+        for (Car car : cars) {
+            int num = generator.generate();
+            car.moveByCondition(num);
+            headPositionUpdate(car.carPosition());
+        }
+    }
+
+    private List<Car> getHeadPositionCars() {
+        return cars.stream().filter(car -> car.carPosition() == headPositionValue).collect(Collectors.toList());
+    }
```

Comment:
```suggestion
    private List<Car> getHeadPositionCars() {
        return cars.stream()
                .filter(car -> car.carPosition() == headPositionValue)
                .collect(Collectors.toList());
    }
```
체이닝식으로 사용하는 `.`은 한 줄에 하나만 사용합시다~!